pages/common/tr.md

```diff
@@ -1,31 +1,27 @@
 # tr
 
-> Translate characters: run replacements based on single characters and character sets.
+> Translate characters: the tr utility will translate (i.e., substitute or delete) charaters from an input stream and output those results.
```

Comment:
I think the original string was fine. replacements already mean substitute or delete. "Input stream" is implicit, and so is "output those results".